### DISCUSSION TITLE
Is there any way to specify the number of axis labels instead of the MinStep?

### DISCUSSION THREAD

**Discussion by ricbrue** [Live-Charts/LiveCharts2#819] [Q&A]
Hi, I have the following chart with two Y axes.

![image](https://user-images.githubusercontent.com/69042836/208101477-0ff6f7cf-86a6-414f-9c05-bacd181a159b.png)

As you can see, I have two different series with two different measure units (the red one is temperature in °C and the blue one is the humidity in %). I would like to have the same number of Y axes labels in both the left and right axes. Is there any way to achieve this? Let's suppose that the temperature could go below zero and so on, so I cannot specify a Min and Max values for the temperature (which is the approach that I have used so far as you can see in the image).

**Comment by beto-rodriguez** [comment]
The docs are not updated to the latest version (it will soon), The library can create better multi axes plots:

![image](https://user-images.githubusercontent.com/10853349/208114393-5c464a67-b482-4278-bd3c-b9fe62dcb4cf.png)

This is easier to read, we had this problem, and we analyzed a lot of options, it seems that adding a "ruler" is a good way to make this clear for the user and also practical for the library.

You can find the code here:

https://github.com/beto-rodriguez/LiveCharts2/blob/master/samples/ViewModelsSamples/Axes/Multiple/ViewModel.cs

But to answer your question, yes you can force the separators, there is the "ForceToMinStep" property, this way you can decide the number of separators.
